src/sap.ui.fl/src/sap/ui/fl/initial/_internal/FlexConfiguration.js

```diff
@@ -36,7 +36,7 @@ sap.ui.define([
 			const vFlexibilityServices = oWritableConfig.get({
 				name: "sapUiFlexibilityServices",
 				type: (vValue) => {
-					if (typeof vValue === "string") {
+					if (vValue && typeof vValue === "string") {
 						if (vValue[0] === "/") {
```

Comment:
How can vValue be "not defined" in the sense of a ReferenceError here? Isn't it defined as the 1st argument of that lambda function? And aren't you allowed to call `typeof` on any value including undefined?

Reply:
the problem is not with typeof.. the problem is with JSON.parse(""). It raises the error and breaks page loading.

Reply:
It's relatively easy to reproduce. Just clone this project, run it with `cds watch` and open http://localhost:4004/travel_analytics/webapp/index.html. With the most recent ui5 version it will not work. Downgrading to 119 helps

Reply:
Actually I just found the reason. It's using `data-sap-ui-flexibilityServices=""` in the html. The point is - this construction is also used in the project too: https://sourcegraph.com/search?q=context%3Aglobal+data-sap-ui-flexibilityServices%3D%22%22&patternType=standard&sm=1&groupBy=repo. But good point - the scope of this issue is very small. So may be we need to update docs giving certain requirements to this parameter. Ideallly it should check like if it's parseable then parse as json otherwise do something else..

Reply:
> JSON.parse("")
Uncaught SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at <anonymous>:1:6

That's a whole different error, right? I do not have the time to reproduce anything, sorry!